app.js

```diff
@@ -83,17 +87,25 @@ const sessionOpts = Object.assign({
 
 app.use(session(sessionOpts));
 
+// check for cookies
+app.use(require('hof').middleware.cookies());
+
 // apps
 app.use(require('./apps/rtm/'));
 
 app.get('/cookies', (req, res) => res.render('cookies'));
 app.get('/terms-and-conditions', (req, res) => res.render('terms'));
 
+// 404's
+app.use(require('hof').middleware.notFound({
+  logger: require('./lib/logger'),
+  translate: i18n.translate.bind(i18n),
```

Comment:
@JoeChapman we need to wait for translations to load before we try and use them, a solution could potentially be to wrap the entire file in an init function and call it on i18n ready